### PR TITLE
partial of #1812 (lsttokens)

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/plugin/lsttokens/FollowersLst.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ public ParseResult parseToken(LoadContext context, CDOMObject obj,
 				+ " on an Ungranted object type: "
 				+ obj.getClass().getSimpleName(), context);
 		}
-		if ((value == null) || (value.length() == 0))
+		if ((value == null) || (value.isEmpty()))
```
Comment (thpr) [inline code/src/java/plugin/lsttokens/FollowersLst.java]: Unnecessary parens around isEmpty

File: code/src/java/plugin/lsttokens/SpellsLst.java [hunk 2]
```diff
@@ -85,7 +85,7 @@ protected ParseResult parseNonEmptyToken(LoadContext context,
 				+ " on an Ungranted object type: "
 				+ obj.getClass().getSimpleName(), context);
 		}
-		if ((sourceLine == null) || (sourceLine.length() == 0))
+		if ((sourceLine == null) || (sourceLine.isEmpty()))
```
Comment (thpr) [inline code/src/java/plugin/lsttokens/SpellsLst.java]: Unnecessary parens around isEmpty

File: code/src/java/plugin/lsttokens/deprecated/ModifyfeatchoiceToken.java [hunk 3]
```diff
@@ -155,7 +155,7 @@ public void applyChoice(CDOMObject owner, CNAbility choice, PlayerCharacter pc)
 		//
 		// If nothing to choose, or nothing selected, then leave
 		//
-		if ((available.size() == 0) || (currentSelections == 0))
+		if ((available.isEmpty()) || (currentSelections == 0))
```
Comment (thpr) [inline code/src/java/plugin/lsttokens/deprecated/ModifyfeatchoiceToken.java]: Unnecessary parens around isEmpty

File: code/src/java/plugin/lsttokens/gamemode/rollmethod/MethodToken.java [hunk 4]
```diff
@@ -65,7 +65,7 @@ protected ParseResult parseNonEmptyToken(LoadContext context,
 	public boolean process(LoadContext context, RollMethod rm)
 	{
 		String method = rm.getMethodRoll();
-		if ((method == null) || (method.length() == 0))
+		if ((method == null) || (method.isEmpty()))
```
Comment (thpr) [inline code/src/java/plugin/lsttokens/gamemode/rollmethod/MethodToken.java]: Unnecessary parens around isEmpty

File: code/src/java/plugin/lsttokens/gamemode/tab/ContextToken.java [hunk 5]
```diff
@@ -42,7 +42,7 @@ public String getTokenName()
 	@Override
 	public ParseResult parseToken(LoadContext context, TabInfo ti, String value)
 	{
-		if ((value == null) || (value.length() == 0))
+		if ((value == null) || (value.isEmpty()))
```
Comment (thpr) [inline code/src/java/plugin/lsttokens/gamemode/tab/ContextToken.java]: Unnecessary parens around isEmpty

**Comment by grimreaper** [comment]
updated

**Pull request merged**